pkg/controller/service/servicecontroller.go

```diff
@@ -590,42 +590,6 @@ func portEqualForLB(x, y *api.ServicePort) bool {
 	return true
 }
 
-func intSlicesEqual(x, y []int) bool {
```

Comment:
Looks like intSlicesEqual is unused, so fine to remove that, but I suggest we just revert the changes in this file so I can justify it for 1.3 (this isn't under cloudprovider/aws)